srcpkgs/python3-BeautifulSoup4/template

```diff
@@ -1,20 +1,21 @@
 # Template file for 'python3-BeautifulSoup4'
 pkgname=python3-BeautifulSoup4
-version=4.10.0
+version=4.11.1
 revision=1
 wrksrc="beautifulsoup4-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3 python3-soupsieve"
 checkdepends="python3-pytest $depends"
 short_desc="Python3 HTML/XML parser"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Ishaan Bhimwal <ishaanbhimwal@protonmail.com>"
```

Comment:
I asked you on IRC to remove adoption from this and your other PRs, please do it and this can be merged.